### PR TITLE
Update names and descriptions

### PR DESCRIPTION
**Pull request by strangeways**
Changes some names and adds descriptions to help identify Controlshift-related things in AWS.

### CODE REVIEW

File: config_table.tf [hunk 1]
```diff
@@ -118,12 +118,12 @@ resource "aws_kms_ciphertext" "redshift_password" {
 }
 
 resource "aws_kms_alias" "lambda_alias" {
-  name = "alias/LambaRedshiftLoaderKey"
+  name = "alias/ControlshiftLambaRedshiftLoaderKey"
```
Comment (woodhull) [inline config_table.tf]: Have you tested that this works? I think this is defined in the upstream code re: how it reads the KMS encryption info.

**Comment by woodhull** [comment]
Another strategy we've had a lot of luck with on AWS is tagging things with the env and app they are related to.

File: config_table.tf (hunk 1 above)
Comment (strangeways) [reply config_table.tf]: Ah, yes, you are correct.

**Comment by woodhull** [comment]
Lets let this run and make sure there are not any other issues in practice before merging tmrw?

**Comment by strangeways** [comment]
Well I definitely screwed something up. 💩 Getting this from the API Gateway even after changing things back: `Execution failed due to configuration error: Invalid permissions on Lambda function`. I'll look at it more in the morning.

**Comment by strangeways** [comment]
Okay, things are working again, though I'm still not sure what happened. I undid some of the name changes, so this PR only modifies things that really shouldn't be a problem. In any case, further testing may be needed to see if this works on a completely new setup.

**Comment by strangeways** [comment]
@woodhull I tested this some more, and it works! The only thing I had to do manually was redeploy the API through the API Gateway web console (under controlshift-webhook-handler -> Resources -> Actions) so that it would trigger the lambda with the updated name (`controlshift-webhook-handler` instead of `recieve-webhook-handler`).

**Pull request merged**